### PR TITLE
Added Business Index, App.js fetch methods,

### PR DESCRIPTION
**Pull request by ecostrong**
While working with our mentor we got ahead of ourselves on this branch. We started working on the business new component. We stopped when we realized to push our changes first. Our create method is next.

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -23,13 +24,38 @@ class App extends React.Component {
...
+      console.log("index errors:", errors)
+    })
+  }
+
+  handleCreateBusiness = (newBusiness) => {
+    console.log(("We did it"))
+  }
+
   render () {
+    // console.log(this.state.businesses)
```
Comment (elysemontano) [inline app/javascript/components/App.js:48]: Please remove this before merging

File: app/javascript/components/pages/BusinessNew.js [hunk 2]
```diff
@@ -1,9 +1,39 @@
...
+  }
+
+  handleChange = (e) => {
+    const {form} = this.state
+    form[e.target.name] = e.target.value
+    this.setState({form: form})
+  }
+  
   render() {
+    console.log(this.state.form.business_name)
```
Comment (elysemontano) [inline app/javascript/components/pages/BusinessNew.js:21]: Let's remove this console log as well.

**Review by elysemontano — APPROVED** [review]
Looks good.  Thank you for updating me with your initial comment on this PR.  Just try to remove any console logs not needed before merging.

File: app/javascript/components/App.js (hunk 1 above)
Comment (dustin-oz) [reply app/javascript/components/App.js:48]: Thank you for pointing this out. We definitely took the approach of just making sure we can pass data in via props and see it returned on screen, and then moved on and planned to come back and clean up and style later..

File: app/javascript/components/pages/BusinessNew.js (hunk 2 above)
Comment (dustin-oz) [reply app/javascript/components/pages/BusinessNew.js:21]: Will do

**Pull request merged**